tests/integration/modules/FetchVerifiedJobTranslationsFromConnectorCest.php

```diff
@@ -403,21 +403,21 @@ public function testExecuteInProgress(IntegrationTester $I): void
             $job->id,
             $element->id,
             'es-ES'
-            //, 703695
+        //, 703695
```

Comment:
why are these lines commented out?

Reply:
They are commented because `assertTranslationInProgress` not having 4 argument now 

will remove this lines, thanks!

Reply:
removed!